docs/user/gcp/apis.md

```diff
@@ -0,0 +1,32 @@
+# GCP Service APIs
+To install OpenShift to your GCP project, certain Service APIs need to be enabled in your project *before* the openshift-installer can install the cluster
+
+## Enable API Services needed by your cluster
+
+You will need to enable the following API services in your project:
+
+- Compute Engine API (`compute.googleapis.com`)
+- Google Cloud APIs (`cloudapis.googleapis.com`)
+- Cloud Resource Manager API (`cloudresourcemanager.googleapis.com`)
+- Google DNS API (`dns.googleapis.com`)
+- Identity and Access Management (IAM) API (`iam.googleapis.com`)
+- IAM Service Account Credentials API (`iamcredentials.googleapis.com`)
+- Service Management API (`servicemanagement.googleapis.com`)
+- Service Usage API (`serviceusage.googleapis.com`)
+- Google Cloud Storage JSON API (`storage-api.googleapis.com`)
+- Cloud Storage (`storage-component.googleapis.com`)
+
+You can enable these services using the console or the CLI (console service names in parentheses)
+
+```sh
+gcloud services enable compute.googleapis.com cloudapis.googleapis.com \
```

Comment:
we don't try to include these examples in the docs because we have to maintain the invocation, and using Cloud docs better provides details, alternatives, and more information and links around these.